efficientdet/det_model_fn.py

```diff
@@ -142,7 +142,24 @@ def learning_rate_schedule(params, global_step):
   raise ValueError('unknown lr_decay_method: {}'.format(lr_decay_method))
 
 
-def focal_loss(y_pred, y_true, alpha, gamma, label_smoothing, normalizer):
+def legacy_focal_loss(logits, targets, alpha, gamma, normalizer, _):
```

Comment:
I kept the original focal loss here for now, such that we can switch the loss function if we see any issues in near future. 

We can remove this legacy_focal_loss in future after we fully verified the new loss works well as expected.

Reply:
:+1: